Sources/SwiftProtobuf/AnyMessageStorage.swift

```diff
@@ -99,7 +99,7 @@ internal class AnyMessageStorage {
     get {
       switch state {
       case .binary(let value):
-        return value
+        return Data(value)
```

Comment:
Can we track it somewhere that we still need to clean this up? Maybe with a TODO or adding it to an issue.

Reply:
I've actually already removed this in the follow-up PR I'll send. I can still add the TODO if you think it's useful?